Dangerfile

```diff
@@ -0,0 +1,41 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
```

Comment:
The string to check could be just WIP is enough, I think sometimes we accidentally use WIP only without square brackets. I'm sure this step is for safety and it should cover more possible cases.
```suggestion
warn("PR is classed as Work in Progress") if github.pr_title.include? "WIP"
```